### DISCUSSION TITLE
Not able to render a collection table in my custom module

### DISCUSSION THREAD

**Discussion by sarthakgaur** [directus/directus#7210] [Archive]
Earlier in version `9.0.0-rc.72`, I used the following code to render a collection table in my custom module:

```
<component
    class="layout"
    ref="layoutRef"
    :is="`layout-tabular`"
    :collection="`nameOfMyCollection`"
    :readonly="true"
>
</component>
```

I recently updated to version `9.0.0-rc.88`, and I am not able to use this component to render a collection table. Any help will be appreciated! Thanks!

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/7063
